app/component/MessageBarMessage.js

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+
+const heading = (e) => {
+  if (e.type === 'heading') {
+    return (<h2>{e.content}</h2>);
+  }
+  return null;
+};
+
+const span = (e) => {
+  if (e.type === 'text') {
+    return (e.content);
+  }
+  return null;
+};
+
+const a = (e) => {
+  if (e.type === 'a') {
+    return (<a href={e.href}>{e.content}</a>);
+  }
+  return null;
+};
+
+const elements = [heading, span, a];
```

Comment:
We could use https://github.com/acdlite/recompose/blob/master/docs/API.md#componentfromprop for these